third_party/libpg_query/src_backend_parser_scan.cpp

```diff
@@ -482,7 +482,7 @@ struct yy_buffer_state
 	/* Size of input buffer in bytes, not including room for EOB
 	 * characters.
 	 */
-	yy_size_t yy_buf_size;
+	int yy_buf_size;
```

Comment:
This fails a warning in the R builds - we should add a replacement to the `generate_flex.py` to make sure this does not get re-generated as an `int` instead of a `yy_size_t`.

As a side note, you don't need to run `generate_flex.py` unless you change the scanner (`scan.l`) - so changing this file is unnecessary to begin with for this PR.